src/ONE.Models.CSharp/Common/Configuration/ConfigurationTypeConstants.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ONE.Models.CSharp.Common.Configuration
+{
+    public static class ConfigurationTypeConstants
+    {
+        public static class Dashboards
+        {
+            public const string Id = "9a4a09ca-cf6c-40c0-bc21-1b8f5d306e0a";
+            public static readonly Guid Guid = new Guid(Id);
+        }
+        public static class Graphs
+        {
+            public const string Id = "24ecb22b-aee3-4bac-a88d-9e10402363ba";
+            public static readonly Guid Guid = new Guid(Id);
+        }
+        public static class Worksheets
```

Comment:
Minor issue but this is technically a WorksheetView

Reply:
Renamed.